crates/biome_js_syntax/src/expr_ext.rs

```diff
@@ -1836,6 +1838,54 @@ impl TsStringLiteralType {
     }
 }
 
+/// Check if the SyntaxNode is a Negate Unary Expression
+/// ## Example
+/// ```js
+/// !x
+/// ```
+pub fn is_negation(node: &JsSyntaxNode) -> Option<JsUnaryExpression> {
```

Comment:
I was not sure if this should be in `parser` or `syntaxt` crates.

Reply:
`biome_js_syntax` is the good place :)